### DISCUSSION TITLE
Unable to Express "AND NOT" Logic in Grafana JSON Filters

### DISCUSSION THREAD

**Discussion by medahalli** [grafana/grafana#79674] [General]
I'm encountering an issue in Grafana when expressing "AND NOT" logic in JSON filters for a query coming from TheHive's API. Currently, I'm fetching data from TheHive into Grafana, and I want to filter the results by excluding items with a certain tag, for example, "TUNING".

Here's a part of my current JSON query in Grafana:
{
    "query": [
        {
            "_name": "listCase"
        },
        {
            "_name": "filter",
            "_field": "status",
            "_value": "Open"
        },
        {
            "_name": "filter",
            "_field": "tags",
            "_value": "$clientID"
        },
        {
            "_name": "filter",
            "_field": "tags",
            "_value": "NOT TUNING"  // This doesn't seem to work as expected
        },
        {
            "_name": "sort",
            "_fields": [
                {
                    "_createdAt": "desc"
                }
            ]
        }
    ]
}
The issue lies in the "NOT TUNING" part, which doesn't seem to be correctly interpreted. I want to exclude items with the "TUNING" tag from my results. Is there a specific syntax in Grafana or my TheHive data source to express negation in JSON filters?

**Comment by natellium** [comment]
Hey there! We want to let you all know that we are moving to Github Issues. You can read more about the move https://github.com/grafana/grafana/discussions/73424.

This one in particular sounds like a potential bug, if you're still running into this issue please open a [bug report](https://github.com/grafana/grafana/issues/new?assignees=&labels=&projects=&template=0-bug-report.yaml&title=Product+Area%3A+Short+description+of+bug). Sorry for the inconvenience, and thanks for your contributions!
